<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteWindow</class>
 <widget class="QWidget" name="DeleteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eliminar Registro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #f3f4f6;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Â¿Seguro que quieres eliminar el registro?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblEliminarRegistroID">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID: 00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="btnEliminarAceptar">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #22c55e;
	color: #fff;
	padding: 10px;
	border-radius: 5px;
}

QPushButton::hover{
	background-color: #16a34a;
}</string>
        </property>
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEliminarCancelar">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #ef4444;
	color: #fff;
	padding: 15px;
	border-radius: 5px;
}

QPushButton::hover{
	background-color: #dc2626;
}</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
